Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3-alpine
+COPY . /app
+WORKDIR /app
+RUN pip install .
+RUN rm -rf /app
```

Comment:
If you're removing stuff, then I'd like to see a fresh multistage `FROM` where you copy the installed asterisklint. Otherwise you're merely hiding used disk space instead of freeing it.

Reply:
done